testutils/clustermanager/util.go

```diff
@@ -58,3 +59,12 @@ func getClusterLocation(region, zone string) string {
 	}
 	return region
 }
+
+func getZoneFromLoc(location string) string {
+	zone := ""
+	parts := strings.Split(location, "_")
+	if len(parts) > 2 {
+		zone = parts[len(parts)-1]
+	}
+	return zone
+}
```

Comment:
```suggestion
parts := strings.Split(location, "_")
if len(parts) > 2 {
  return parts[len(parts)-1]
}
return ""
}
```

Reply:
somehow it doesn't allow me to commit the change, manually added it